doc/plugin_tutorial/tuto1/src/simple_check.ml

```diff
@@ -1,32 +1,14 @@
-let simple_check1 value_with_constraints =
-  begin
-    let evalue, st = value_with_constraints in
-    let sigma = Evd.from_ctx st in
-(* This is reverse engineered from vernacentries.ml *)
-(* The point of renaming is to make sure the bound names printed by Check
-   can be re-used in `apply with` tactics that use bound names to
-   refer to arguments. *)
-    let j = Environ.on_judgment EConstr.of_constr
-      (Arguments_renaming.rename_typing (Global.env())
-         (EConstr.to_constr sigma evalue)) in
-    let {Environ.uj_type=x}=j in x
-  end
-
-let simple_check2 value_with_constraints =
-  let evalue, st = value_with_constraints in
-  let sigma = Evd.from_ctx st in
+let simple_check1 env (sigma, evalue) =
 (* This version should be preferred if bound variable names are not so
   important, you want to really verify that the input is well-typed,
   and if you want to obtain the type. *)
```

Comment:
Having removed the old check1, the `if bound variable names are not so important` part of the comment is confusing.